classes/class-wc-connect-nux.php

```diff
@@ -315,20 +315,13 @@ public function show_nux_banner( $content ) {
 						<?php echo esc_html( $content['description'] ); ?>
 					</p>
 					<?php if ( $content['should_show_jp'] ) : ?>
-						<p><?php
-						/* translators: %1$s example values include "Install Jetpack and CONNECT >", "Activate Jetpack and CONNECT >", "CONNECT >" */
```

Comment:
This code seems to be from an errant commit - cc4d442bd8b5bae9fa476a4a1167826043985a62. It doesn't look like it should be in this branch.